backend/src/repos/BuildingRepo.js

```diff
@@ -0,0 +1,25 @@
+import QueryHandler from './QueryHandler';
+
+export class BuildingRepo extends QueryHandler {
+
+    constructor(db,errorCodes) {
```

Comment:
@gulyaasjaanos pay attention to styling
```suggestion
    constructor(db, errorCodes) {
```